### PR TITLE
Some minor changes in section computation and plotting

### PR DESCRIPTION
**Pull request by elimh**

### CODE REVIEW

**Comment by Leguark** [comment]
All plot data is broken

File: gempy/core/solution.py [hunk 1]
```diff
@@ -109,14 +109,12 @@ def set_solution_to_regular_grid(self, values: Union[list, np.ndarray], compute_
         return self
 
     def set_solution_to_topography(self, values: Union[list, np.ndarray]):
-        l0, l1 = model.grid.get_grid_args('topography')
-        model.solutions.geological_map = values[0][:, l0: l1]
-        model.solutions.geological_map_scalfield = values[3][:, l0: l1].astype(float)
+        l0, l1 = self.grid.get_grid_args('topography')
```
Comment (Leguark) [inline gempy/core/solution.py]: This seem as a good idea

File: gempy/core/data.py [hunk 2]
```diff
@@ -698,7 +698,7 @@ def update_colors(self, cdict=None):
         Returns: None
 
         '''
-        if cdict == None:
+        if not cdict:
```
Comment (Leguark) [inline gempy/core/data.py]: Usually the best practice is if cdict is None. Is the less ambiguous condition and gives less problems down the line

File: gempy/core/gempy_api.py [hunk 3]
```diff
@@ -333,16 +333,11 @@ def compute_model(model: Model, output=None, compute_mesh=True, reset_weights=Fa
         if model.grid.active_grids[0] is np.True_:
             model.solutions.set_solution_to_regular_grid(sol, compute_mesh=compute_mesh)
         if model.grid.active_grids[1] is np.True_:
-            l0, l1 = model.grid.get_grid_args('custom')
-            model.solutions.custom = sol[0][:, l0: l1]
+            model.solutions.set_solution_to_custom(sol)
```
Comment (Leguark) [inline gempy/core/gempy_api.py]: Thanks for reworking this

File: gempy/core/grid_modules/grid_types.py [hunk 4]
```diff
@@ -141,38 +147,13 @@ def compute_section_coordinates(self):
...
-            # abs needed for cases where phi == -1/4 pi or 3/4 pi
-            if x0 > x1:
-                n_points = np.ceil((x0 - x1) / (ds * np.cos(phi)))
-            else:
-                n_points = np.ceil((x1 - x0) / (ds * np.cos(phi)))
-            xi = np.linspace(x0, x1, int(n_points))
-            m = (y1 - y0) / (x1 - x0)  # slope of line
-            yj = m * (xi - x0) + y0 * np.ones(xi.shape)  # calculate yvalues with line equation
-        return np.vstack((xi, yj)).T
+    def calculate_line_coordinates_2points(self, p1, p2, res):
```
Comment (Leguark) [inline gempy/core/grid_modules/grid_types.py]: This refactor is impressive :D

File: gempy/utils/create_topography.py [hunk 5]
```diff
@@ -30,6 +30,10 @@ def __init__(self, path_dem, grid=None):
         if GDAL_IMPORT == False:
             raise ImportError('Gdal package is not installed. No support for raster formats.')
         self.dem = gdal.Open(path_dem)
+
```
Comment (Leguark) [inline gempy/utils/create_topography.py]: These checks are also great

File: gempy/core/data.py (hunk 2 above)
Comment (elimh) [reply gempy/core/data.py]: Oh okay, I changed this because the code review said that is the proper way to do it

**Pull request merged**